.vscode/launch.json

```diff
@@ -0,0 +1,28 @@
+{
+  "version": "0.2.0",
```

Comment:
_(arbitrary comment location to start a thread)_

I have WETH in my wallet but am unable to do the shielding. I sign the message, but never get the approve or shield tx prompts. Instead I get this error, using metamask + goerli

![image](https://user-images.githubusercontent.com/17163988/212189027-38d40e9b-cb10-486c-9fdc-de658877eba1.png)

Reply:
the approve is currently commented out

Reply:
Can you give more context here? Should that be uncommented out in this PR, or are we handling that in a follow up PR somewhere? (took a quick look and don't see an issue for approvals)

Reply:
i'm uncommenting it, and we will track a more elegant `approve` in #42

Reply:
I know you have #42 for more elegant approve, but so far I haven't successfully shielded anything on this branch which is why I haven't approved yet 😅

Reply:
Now that my approval tx went through, I went through the flow again and it does work now! So I guess we can mark this resolved? Will defer to you, but if so consider this approved from me